community/modules/scheduler/schedmd-slurm-gcp-v5-hybrid/main.tf

```diff
@@ -22,13 +22,19 @@ locals {
   install_dir         = var.install_dir != null ? var.install_dir : abspath(var.output_dir)
   install_dir_pattern = replace(local.install_dir, ".", "\\.")
   install_path_cmd    = "sed -i -E 's|Program=/.*/(resume\\|suspend).py|Program=${local.install_dir_pattern}/\\1\\.py|g' cloud.conf"
+
+  # Since deployment name may be used to create a cluster name, we remove any invalid character from the beginning
+  # Also, slurm imposed a lot of restrictions to this name, so we format it to an acceptable string
+  tmp_cluster_name   = substr(replace(lower(var.deployment_name), "/^[^a-z]*|[^a-z0-9]/", ""), 0, 10)
```

Comment:
nit: I think `default_cluster_name` would be clearer.

Reply:
I agree, however `tmp_cluster_name` is in line with the controller and login implementation, so I'm going to keep it as is to avoid having the 3 diverge.